server/webapp/WEB-INF/vm/auth/login.vm

```diff
@@ -123,6 +123,11 @@
                 </div>
                 #end
                 <div class='clear-both'><!-- Clear floats --></div>
+
+#foreach ($authentication_plugin_id in $authentication_plugin_registry.getAuthenticationPlugins())
```

Comment:
But, will Go's default login page be allowed to change? If not, why shouldn't it? Why shouldn't an admin be able to say, "All our auth has to happen through blah plugin. So, the default login page should redirect to that". Feasible? Makes sense, or not?